doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,151 @@
+   * Info trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_INF`
+   * Debug trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_DBG`
+
+See :ref:`zigbee_ug_logging_stack_logs` to read more about trace logs.
+
+
+
+.. rst-class:: numbered-step
+
+Configure how to print ZBOSS trace logs
+***************************************
+
+The :ref:`zigbee_osif_zboss_osif_serial` offers a few backends to choose from for printing ZBOSS trace logs.
+It is recommended to use the Zigbee serial logger, as it is the most efficient.
+To enable it, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_BINARY_LOGGING`.
+
+**Optional:** You can increase size of the ring buffer that temporarily stores the trace logs.
+   * To do this, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_BUFFER_SIZE`.
+   * This can prevent losing some of the logs in demanding scenarios such as high network traffic, multiple devices being configured or joined, and so on.
+
+
+Trace logs using UART (default)
+===============================
+
+Trace logs using the UART are enabled by default.
+Below are more details on how to configure trace logs using the UART.
+
+1. Trace logs are enabled by default. However, it can be manually set with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_UART_LOGGING`.
+
+#. Configure which UART device you want to use with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
+   This is optional, as the default ``UART_1`` will be used if no other UART device is configured.
+
+#. Configure ``UART_1`` to be connected to the onboard J-Link instead of ``UART_0``, by extending the DTS overlay file for the selected board with the following:
+
+    .. code-block:: devicetree
+
+      &uart1 {
+         tx-pin = <6>;
+         rx-pin = <8>;
+         rts-pin = <5>;
+         cts-pin = <7>;
+      };
+
+      &uart0 {
+         rx-pin = <33>;
+         tx-pin = <34>;
+         /delete-property/ rts-pin;
+         /delete-property/ cts-pin;
+      };
+
+   .. note::
+      This will remove access to the application logs.
+
+
+Optional configuration
+-----------------------
+
+You can also increase the UART throughput by changing the baudrate.
+Some of the trace logs will be dropped if the throughput is too low.
+By default, the UART baudrate is set to 115200.
+To increase the baudrate to ``1000000``, add the ``current-speed = <1000000>;`` property to the ``uart1`` node in the DTS overlay file.
+
+
+
+Trace logs using native USB
+===========================
+
+Trace logs can also be configured using a native USB.
+
+
+1. Set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING`.
+   This also enables the necessary USB Kconfig options.
+
+#. Configure which USB device to use with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
+   This is optional, as the default ``CDC_ACM_0`` will be used if no other USB device is configured.
```

Comment:
```suggestion
#. Optionally, configure which USB device to use with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
   The default ``CDC_ACM_0`` will be used if no other USB device is configured.
```

Reply:
Please double-check this line, as the preview looks like not updaed, and the line in the file is also not updated by the looks of it in GitHub...